docs/DocGenerator/README.md

```diff
@@ -0,0 +1,205 @@
+
+Each test file has self-contained documentation comment blocks in **YAML** format.
+
+These blocks can contain **Mandatory**, **Optional**, and **Ignored** fields.
+In the [Configuration section](#configuration) it's explained how to configure these fields.
+- **Mandatory** fields must be present in the documentation block and will be dumped into the final
+documentation output.
+- **Optional** fields, if present, will be parsed and dumped into the final documentation output.
+- **Ignored** fields are the ones not listed in the configuration file. They can be added by the developer to explain
+a functionality of the module or a method without transferring this information to the final documentation output.
+
+A test file has a module documentation block at the top level of the file with information related to the complete
+test file. And each test method inside the file will have a test documentation block with the specific information of
+this test.
+
+Additional group information is parsed from the README files found in the repository. Each README file represents
+a group, and every test file at the same or lower folder level than this group is considered as belonging to this group.
+
+Also, groups could be nested and a README found under the level of another group will generate a new group that belongs
+to the first one and contains any group or tests under it.
+
+The specific content of each block is defined in the [Documentation schema section](#documentation-schema)
+
+### Parsing
+Running DocGenerator as specified in the [Usage section](#usage) will scan every test and group file found into the
+include paths of the documentation, it will extract the module and tests blocks from each test file and discard any
+non-documentable field. Also, complementary test-cases information will be extracted from a dry-run of pytest if there
+isn´t a description for them.
+
+### Output
+The parsed and filtered documentation information will be dumped in the Output folder defined by configuration.
+
+Each test file will generate a JSON and a YAML file with the documentation information. Each of these files contains
+a structure with the module description and every test function in the module. Each test function
+contains its description and, if available, the test-cases information.
+
+These files are ordered in nested folders with the same tree structure as the source code files.
+
+### Sanity Check
+After the generation of the documentation output, a sanity check can be executed to identify the coverage, tags and
+any missing mandatory field in the documentation output.
+
+### Indexing
+The documentation output is intended to be indexed and displayed with SearchUI App. So, DocGenerator has the capability
+to automatically generate the index of the parsed information into React. This will load all the content of the output
+folder with a specific index name. Making them available for visualization.
```

Comment:
I don't think this paragraph is clear enough. It should mention that the information in JSON format is added to a `elasticsearch` index and `Search-UI` visualize the information by querying `elasticsearch`

Reply:
I think that the last part isn´t related with the Indexing stage, maybe with the Local Lunch stage. Would you suggest rephrasing **Local Lunch** section?